apps/arweave/src/ar_tx.erl

```diff
@@ -8,6 +8,7 @@
 -export([generate_chunk_tree/1, generate_chunk_tree/2, generate_chunk_id/1]).
 -export([chunk_binary/2]).
 -export([chunks_to_size_tagged_chunks/1, sized_chunks_to_sized_chunk_ids/1]).
+-export([tx_format_to_http_header/1, tx_http_format/1]).
```

Comment:
Looks like `tx_http_format/1` is used inside of current module only - no need to export

Reply:
True. However, `tx_http_format/1` should be a usable utility for translating from an internal tx-format to the external http format (for internal purposes as well). It's not always the case that this function will only be used for creating an HTTP header via `tx_format_to_http_header/1`.